extensions/amp-access/0.1/amp-access.js

```diff
@@ -125,6 +139,78 @@ export class AccessService {
   startInternal_() {
     actionServiceFor(this.win).installActionHandler(
         this.accessElement_, this.handleAction_.bind(this));
+
+    // Start authorization XHR immediately.
+    this.runAuthorization_();
+  }
+
+  /**
+   * @return {!Promise}
+   * @private
+   */
+  runAuthorization_() {
+    log.fine(TAG, 'Start authorization via ', this.config_.authorization);
+    this.toggleTopClass_('amp-access-loading', true);
+
+    // TODO(dvoytenko): produce READER_ID and create the URL substition for it.
```

Comment:
can we create an issue for this? (guessing its just another `set_` call on url replacements?)